gtfs-realtime/spec/en/trip-updates.md

```diff
@@ -47,7 +47,43 @@ The information provided by the trip descriptor depends on the schedule relation
 | **Unscheduled** | This trip is running and is never associated with a schedule. For example, if there is no schedule and the buses run on a shuttle service. |
 | **Canceled** | This trip was scheduled, but is now removed. |
 
-In most cases, you should provide the trip_id of the scheduled trip in GTFS that this update relates to. In case you are not able to link this update to a trip in GTFS, you can instead provide a GTFS route_id and a start time and date for the trip. This is normally the case for added, unscheduled or some types of replacement trips.
+In most cases, you should provide the trip_id of the scheduled trip in GTFS that this update relates to. 
+
+#### Systems with repeated trip_ids
+
+For systems using repeated trip_ids, for example trips modeled using frequencies.txt, that is frequency-based trips, the trip_id is not in itself a unique identifier of a single journey, as it lacks a
+specific time component. In order to uniquely identify such trips within a
+TripDescriptor, a triple of identifiers must be provided:
+
+*    __trip_id__
+*    __start_time__
+*    __start_date__
+
+start_time should be first published, and any subsequent feed updates should use
+that same start_time when referring to the same journey. StopTimeUpdates
+should be used to indicate adjustments; start_time does not have to be precisely
+the departure time from the first station, although it should be pretty close to
+that time.
+
+For example, let’s say we decide at 10:00, May, 25th 2015, that a trip with
+trip_id=T will start at start_time=10:10:00, and provide this information via
+realtime feed at 10:01. By 10:05 we suddenly know that the trip will start not
+at 10:10 but at 10:13. In our new realtime feed we can still identify this trip
+as (T, 2015-05-25, 10:10:00) but provide a StopTimeUpdate with departure from
+first stop at 10:13:00.
```

Comment:
The conversation @barbeau describes is in the comments on this document (along with some other points which may be relevant to this issue): https://docs.google.com/document/d/13pkZPVxWcphCLzSO-CpetUZ4onqZ0BN11oIpf78gNp4/edit#heading=h.fy8gzwwva0d4

I believe this is what WMATA does with some trips on the airport buses during the holidays.